src/main/webapp/features.jsp

```diff
@@ -2,186 +2,217 @@
 <t:staticPage currentPage="features">
   <div class="container">
     <div class="lead">
-      Here is an overview of some notable TEAMMATES features.
-      The <a href="instructorHelp.jsp">help page</a> contains more details about how to use these features.
+      <h1 class="color_orange">
+        Features
+      </h1>
+      <p class="h5"> Here is an overview of some notable TEAMMATES features.
+        The <a href="instructorHelp.jsp">help page</a> contains more details about how to use these features.
+      </p>
     </div>
+
     <section class="row">
       <div class="col-xs-12 col-sm-8 col-sm-offset-2">
         <h2 class="color_orange col-xs-12">Team peer evaluations</h2>
-        <img class="img-responsive col-xs-12" src="images/feature-teampeerevaluations.png" alt="Team peer evaluations">
-        <div class="col-xs-12 col-sm-10 col-sm-offset-1">
-          Have a team project in your course? Set up a 'team peer evaluation session' for students to give anonymous peer feedback to team members.
-          <br><br>
-          Students can provide confidential peer evaluations to you too.
-        </div>
-        <img class="img-responsive" src="images/raised-edge.png">
+        <img class="features-page-img" src="images/feature-teampeerevaluations.png" alt="Team peer evaluations">
```

Comment:
According to https://www.w3schools.com/bootstrap/bootstrap_ref_css_images.asp, the `img-responsive` class applies `max-width: 100%`, `height: auto`, and `display:block` to the image. Perhaps use both classes (i.e. `class="img-responsive features-page-img"`) here? Then modify `features-page-img` in `teammatesCommon.css` to use `max-width` instead of `width`. If I'm not wrong, setting `width` can cause the image to scale up to greater than its original resolution on large screen resolutions.